### PR TITLE
Simplify the integration test in adding a function to turn on/off an editing tool

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: test/integration/test_utils.mjs [hunk 1]
```diff
@@ -581,6 +581,22 @@ async function kbFocusPrevious(page) {
   await awaitPromise(handle);
 }
 
+async function switchToEditor(name, page, disable = false) {
+  const modeChangedHandle = await createPromise(page, resolve => {
+    window.PDFViewerApplication.eventBus.on(
+      "annotationeditormodechanged",
+      resolve
+    );
```
Comment (Snuffleupagus) [inline test/integration/test_utils.mjs:589]: This listener is never being removed, which probably isn't what you want. However, fixing this just requires adding one line here:
```suggestion
    window.PDFViewerApplication.eventBus.on(
      "annotationeditormodechanged",
      resolve,
      { once: true }
    );
```

**Review by Snuffleupagus — APPROVED** [review]

**Comment by timvandermeij** [comment]
Thank you for doing this refactoring; looks much better/simpler like this!

**Pull request merged**